### PR TITLE
Get true dimensions for image metadata

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
Image metadata dimensions were based on the base image asset dimensions, but really should have been the true image dimensions that we actually serve after resizing.  Tidy up after #17258 

## What is the value of this and can you measure success?
Image metadata dimensions are accurate.  I don't know whether this has much effect, because I'm not sure how much these numbers are used by search engines etc.

## Does this affect other platforms - Amp, Apps, etc?
Yes

## Tested in CODE?
No

### CODE REVIEW

File: article/app/model/structuredData/Image.scala [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-import play.api.libs.json.{JsString, JsValue, Json}
+import model.ImageElement
+import play.api.libs.json.{JsValue, Json}
 import views.support.Item700
 
 object Image {
 
   def apply(picture: ImageElement): JsValue = {
-    val asset: Option[ImageAsset] = Item700.bestFor(picture.images)
+    val url = Item700.bestSrcFor(picture.images).getOrElse("")
```
Comment (jfsoul) [inline article/app/model/structuredData/Image.scala]: had to define these here to make the compiler happy 😨

**Review by NataliaLKB — APPROVED** [review]

**Comment by jfsoul** [comment]
Actually it looks like I need to take the minimum of the original asset dimension and the (attempted) resize by imgix.  Imgix won't resize to any larger than the original dimensions as far as I can tell.  cc: @paperboyo does that sound right?

**Comment by paperboyo** [comment]
> Imgix won't resize to any larger than the original dimensions as far as I can tell

Yep, that’s right (`fit=max` added [here](https://github.com/guardian/frontend/pull/12079#issue-136091332)). Nor should it (quality wouldn’t be any better, filesize adversely affected)!

**Comment by jfsoul** [comment]
Thanks for people's time looking at this for me, but I'm taking the pragmatic decision and leaving this unchanged for now.  I've fixed up the logic to be closer to the true behaviour, but until someone can prove this metadata actually affects anything I don't want to introduce this unnecessary complexity to the codebase.

N.B. I still don't think my most recent attempt is quite right because it assumes `fit=max` resizing is always width-driven, when height could also the constraining factor.

**Pull request closed without merging**